core/src/test/java/io/grpc/internal/ClientCallImplTest.java

```diff
@@ -120,6 +124,109 @@ public void advertisedEncodingsAreSent() {
     assertEquals(decompressorRegistry.getAdvertisedMessageEncodings(), acceptedEncodings);
   }
 
+  @Test
+  public void prepareHeaders_authorityAdded() {
```

Comment:
In modern JUnit4 tests, yes.  Several other readability reviews have been recommending it for a while.